lib/report/system.go

```diff
@@ -151,11 +151,7 @@ cat %v 2> /dev/null || true`
 // planetLogs fetches planet syslog messages as well as the fresh journal entries
 func planetLogs(since time.Duration) Collectors {
 	return Collectors{
-		// Fetch planet journal entries for the last two days
```

Comment:
This will effectively prevent recovering structured journal logs - I thought the agreement was that we **add** plain-text logs in addition to structured ones - @r0mant @bernardjkim?

Reply:
Oh, we can include both formats. We'll keep the `planet-journal-export.log` and add a plain text `planet-journal.log`. We'll do the same for the gravity journal logs as well.